test/e2e/data/infrastructure-docker/main/clusterclass-quick-start.yaml

```
@@ -447,9 +522,17 @@ spec:
         nodeDrainTimeout: 1s
       kubeadmConfigSpec:
         clusterConfiguration:
+          # extraArgs must be non-empty for control plane components to enable patches from ClusterClass to work.
           controllerManager:
-            extraArgs: { enable-hostpath-provisioner: 'true' }
+            extraArgs:
+             enable-hostpath-provisioner: 'true'
+             v: "0"
+          scheduler:
+            extraArgs:
+             v: "0"
```

```suggestion
            extraArgs:
              enable-hostpath-provisioner: 'true'
              v: "0"
          scheduler:
            extraArgs:
              v: "0"
```
indent seems wrong here?